### DISCUSSION TITLE
Using wrong version of Blender needs error message and clearer documentation

### DISCUSSION THREAD

**Discussion by AaronOldenburg** [godotengine/godot-proposals#9409] [3D]
When you set up Godot to directly import .blend files and put a blender file in your project, if you aren't using the right version of Blender, the file doesn't show up but there is no error.

This is a pretty small change, but I don't see any "needs error message" bug reports, so I think it falls under feature requests. If I'm wrong, I'll move it to bug reports.

Another issue is that this happened to me with Blender 3.0.1 installed, and I read somewhere that at least Blender 3.3 is required, but all the hover text and other documentation in Godot says you only need Blender 3.0.

Here is my setup:

  Project Settings -> FileSystem -> Import 
  Blender is Enabled
  
  Editor Settings -> FileSystem -> External Programs
  3D Model Editor path set to "/bin/blender"

v4.2.1.stable.official gives this error:

> modules/gltf/register_types.cpp:63 - Blend file import is enabled in the project settings, but no Blender path is configured in the editor settings. Blend files will not be imported.

Even though the path to Blender is specified.

It looks like that's been somewhat fixed in v4.3.dev... it removes the confusing error, and just doesn't give any.

My suggestions are:

1. Give an error message when it's set up to import Blend files and the Blender program location is specified, but it's the wrong version of Blender
2. Update documentation (and compatibility checks) to state that something higher than Blender 3.0, like 3.3, is required, if that is indeed the case.

**Comment by AThousandShips** [comment]
> Another issue is that this happened to me with Blender 3.0.1 installed, and I read somewhere that at least Blender 3.3 is required

Where did you hear this? Have you confirmed it doesn't work? Looking at the importer 3.0 is the minimum required version 

Also the path in 4.2+ is `filesystem/import/blender/blender3_path`, did you set it? You talk about the external editor path, that's different

**Reply by AaronOldenburg** [reply]
OK, I had the editor setting wrong. When I initially went into the Editor Settings and did a filter for "blender" nothing came up (or at least it didn't in Godot 4.2), so I typed "3D" and got the 3D modeler external program path, so assumed it was that. I just now did a manual search for the blender path under Import and found the correct setting. Now it also comes up when I filter for "blender"... don't know if that's something that was fixed in 4.3, or it works now that I manually found it... which would be odd behavior. Of course, now that the path is in the correct place, it also successfully imports the blender files.

**Reply by AaronOldenburg** [reply]
Thanks
